src/a01/docker/droid_image.py

```diff
@@ -1,108 +0,0 @@
-import functools
-import json
-import re
-from typing import List, Optional
-
-import docker
-import docker.errors
-import docker.models.images
-
-from a01.common import get_logger
-
-
-class DroidImage(object):
```

Comment:
All this is already happening in the dispatcher IIRC?

Reply:
Yes.